internal/config/config.go

```diff
@@ -28,14 +29,8 @@ type AppConfig struct {
 	AdminPassword   string `yaml:"adminPassword"`
 	// Port sets the port that the web UI will listen on.
 	// Defaults to 8000
-	Port    int `yaml:"port"`
-	Storage struct {
-		// Directory that VPN devices (WireGuard peers)
-		// should be saved under.
-		// If this value is empty then an InMemory storage
-		// backend will be used (not recommended).
-		Directory string `yaml:"directory"`
-	} `yaml:"storage"`
+	Port      int                    `yaml:"port"`
+	Storage   storage.StorageWrapper `yaml:"storage"`
```

Comment:
I think it's simpler to just use a string here. This is a bit too fancy I think.

Reply:
I liked the wrapper solution cause it meant that the logic was in a single place. I'm open to other solution ideas though